rdt/transformers/numerical.py

```diff
@@ -102,12 +102,11 @@ def fit(self, data):
             data = pd.Series(data)
 
         self._dtype = self.dtype or data.dtype
-        clean_data = data.dropna()
```

Comment:
Why did we get rid of this? If you do `data.min()` on data that has a `np.nan` value, it will return `np.nan` as the min. Same with max

Reply:
No, `data.min()` and `data.max()` seem to return the right values. Bear in mind that at this point `data` is a `pd.Series`, so it works around the NaNs.

```
In [2]: pd.Series([1, 2, None]).min()
Out[2]: 1.0
```

Reply:
that explains my confusion